src/Command/AddTimeEntryCommand.php

```diff
@@ -23,6 +23,8 @@
 
 class AddTimeEntryCommand extends Command
 {
+    const COMMAND_NAME = 'timetracking:add';
```

Comment:
as the class already extends `Command`, WDYT about `const NAME` ?

Reply:
you just mean use more compact constant name, `NAME` instead of `COMMAND_NAME`?